lib/elixir/test/elixir/module/types/integration_test.exs

```diff
@@ -90,7 +90,7 @@ defmodule Module.Types.IntegrationTest do
 
       warnings = [
         "Kernel.behaviour_info/1 is undefined or private",
-        "a.ex:6: A.e/0"
+        "a.ex:6:20: A.e/0"
```

Comment:
Those are pointing to the dot, right? Is this what we expect?

Reply:
I don't think so, but it's what we're getting :laughing:
Currently warnings in the undefined remote function call point to the last dot instead of the starting module.

If we get to fix this we can also add the span to this case I suppose (similar to the unknown local function call)